R/convert-psn.R

```diff
@@ -21,20 +20,23 @@
 #'
 #' @export
 convert_psn <- function(.modelfit_dir,
-                        .bbr_dir = dirname(.modelfit_dir),
+                        .bbr_dir,
                         .psn_model_file = NULL,
                         .description = NULL,
                         .based_on = NULL,
                         .tags = NULL,
                         .star = NULL,
-                        .cleanup_psn = FALSE,
                         .overwrite = FALSE){
 
+  # Make sure .bbr_dir is not set to .modelfit_dir
+  if(normalizePath(.modelfit_dir) == normalizePath(.bbr_dir)){
+    stop("`.bbr_dir` cannot be set to the same location as `.modelfit_dir`")
+  }
+
 
-  # TODO: Make sure bbi path is set BEFORE copying any files
+  # Make sure bbi path is set BEFORE copying any files
   # this is needed to create a fake bbi json, and we dont want to copy everything just to fail at the end
-  # is_empty wont capture all cases - revisit how to properly address this concern
-  if(rlang::is_empty(bbi_version())){
+  if(!nzchar(bbi_version())){
```

Comment:
maybe I'm missing something but do they actually need to have `bbi` installed to run this? They'll need it to do `model_summary()`, which is a likely next step. But I don't think we actually need `bbi` for this function, right?